src/main/java/com/philderbeast/prisonpicks/Util.java

```diff
@@ -70,7 +72,6 @@ static int calculateExperienceForBlock(Block block)
                 min = 3; 
                 max = 7; 
                 break; 
-            case GLOWING_REDSTONE_ORE:
```

Comment:
why has this been removed? remember case statements without a break fall through

Reply:
believe 1.17.1 removed glowing_redstone_ore id